src/pages/nøkkeltall/index.tsx

```diff
@@ -0,0 +1,93 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { Button, Loader, Panel } from '@navikt/ds-react';
+import { Feilmelding, Ingress, Normaltekst, Element } from 'nav-frontend-typografi';
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router';
+
+import { pipe } from '~/lib/fp';
+import { hentNøkkeltall } from '~api/nøkkeltallApi';
+import Oppsummeringspanel, {
+    Oppsummeringsfarge,
+    Oppsummeringsikon,
+} from '~components/revurdering/oppsummering/oppsummeringspanel/Oppsummeringspanel';
+import { useApiCall } from '~lib/hooks';
+import { useI18n } from '~lib/i18n';
+
+import styles from './index.module.less';
+import messages from './nøkkeltall-nb';
+
+const NøkkelTall = () => {
+    const [nøkkeltallStatus, fetchNøkkeltall] = useApiCall(hentNøkkeltall);
+    const { intl } = useI18n({ messages });
+    const history = useHistory();
+
+    useEffect(() => {
+        fetchNøkkeltall({});
+    }, []);
+
+    const Rad = ({ messageId, verdi }: { messageId: string; verdi: number }) => (
```

Comment:
Flytt gjerne denne komponenten ut av kroppen på den andre, og ta inn selve meldingen i stedet for `messageId`.